Formula/terrahub.rb

```diff
@@ -0,0 +1,33 @@
+require "language/node"
+
+class Terrahub < Formula
+  desc "Terraform automation and orchestration tool (open source)"
```

Comment:
Since open source is a requirement for inclusion in homebrew, I think you can drop that here.

Reply:
Done